### ISSUE TITLE
Support for Nosey.com

### ISSUE THREAD

**Issue by dwalkerdon** [ytdl-org/youtube-dl#23120]
I was Wondering if the team could add support for nosey.com. here is a link example: https://www.nosey.com/watch/channel/nosey_general/14-c3al0cgz1ub4-stunning-secrets-revealed

**Comment by Thrasher88** [comment]
I agree! I found a temporary method to grabbing the streams that will sometimes work, but it's resulted in some glitchy video for me at times. It works more often than not though.
All you need is a browser with Developer tools and VLC Media Player.
Once you grab the master.m3u8 link to the video you want (using Developer tools), open the file using Notepad. There will be several links showing different streams.
Copy the very last link
For example:
'#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=3919000,RESOLUTION=640x480'

Underneath that will be a link.
Copy it, then paste that link in your browser. It will download something called 'Rendition.m3u8'.
Now open up VLC Media Player, select 'Media' at the top left corner, scroll down and select 'Convert / Save'.
Now find your 'Rendition.m3u8' file, select it and hit 'Convert / Save' below.
Now rename the '.m3u8' file to '.mp4'
It will save the highest quality stream copy of the video for you to a .mp4 file so you can watch it on your TV or whatever you want.

Sometimes the file ends up being a little jumpy. Right now I'm dealing with lousy sound quality on some videos, but there is probably a setting that I'm neglecting.
I'm hoping youtube-dl will support this site in the future. If you're into oldschool talkshows, this site is a treasure trove.
